.github/workflows/_kotlin.yml

```diff
@@ -57,20 +81,3 @@ jobs:
         run: |
           echo -n "$FIREBASE_APP_DISTRIBUTION_CREDENTIALS" > $FIREBASE_CREDENTIALS_PATH
           ./gradlew appDistributionUploadRelease uploadCrashlyticsSymbolFileRelease
-
```

Comment:
This seemed to be dead code leftover from #7538

Reply:
This isn't dead code. I use this to get a debug build of the Android app for testing on devices. Building the app locally is an absolute pain on NixOS :(